.spelling

```diff
@@ -885,6 +891,7 @@ Sr4
 twinax
 USR SFP-10G
 vcs-user-credentials
+x86_64
```

Comment:
[This PR](https://github.com/Cray-HPE/docs-csm/pull/3958/files) is using back ticks for `aarch64`.

Instead of adding x86_64, arm64, and aarch64, can they be backticked instead?